src/workflows-app/SubmissionDetails.test.js

```diff
@@ -690,4 +781,304 @@ describe('Submission Details page', () => {
+        })
+      );
+    });
+
+    const executionLogButtons = screen.getAllByRole('button', { name: 'Log' });
+
+    for (const executionLogButton of executionLogButtons) {
+      // Act
+      await user.click(executionLogButton);
+
+      // Assert
+      screen.getByRole('dialog', { name: 'Workflow Execution Log' });
+      screen.getByText('File:');
+      screen.getByText('workflow.log');
+      await screen.findByRole('link', { name: 'Download log' });
+      screen.getByRole('button', { name: 'Workflow Execution Log' });
+      screen.getByText('this is the text of a mock file');
+    }
+  });
+
+  it('should open the task data modal when Inputs button is clicked', async () => {
+    // Arrange
+    const user = userEvent.setup();
+    const getRuns = jest.fn(() => Promise.resolve(mockRunsData));
+    const getRunsSets = jest.fn(() => Promise.resolve(runSetData));
+    const getMethods = jest.fn(() => Promise.resolve(methodData));
+    const mockLeoResponse = jest.fn(() => Promise.resolve(mockAzureApps));
+    Ajax.mockImplementation(() => {
+      return {
+        Cbas: {
+          runs: {
+            get: getRuns,
+          },
+          runSets: {
+            get: getRunsSets,
+          },
+          methods: {
+            getById: getMethods,
+          },
+        },
+        Apps: {
+          listAppsV2: mockLeoResponse,
+        },
+        CromwellApp: {
+          workflows: () => {
+            return {
+              metadata: jest.fn(() => {
+                return Promise.resolve(runDetailsMetadata);
+              }),
+            };
+          },
+        },
+        AzureStorage: {
+          details: jest.fn().mockResolvedValue({ sas: { token: '1234-this-is-a-mock-sas-token-5678' } }),
+        },
+        Metrics: {
+          captureEvent,
+        },
+      };
+    });
+
+    // Act
+    await act(async () => {
+      render(
+        h(BaseSubmissionDetails, {
+          name: 'test-azure-ws-name',
+          namespace: 'test-azure-ws-namespace',
+          workspace: mockAzureWorkspace,
+          submissionId,
+          workflowId: '00001111-2222-3333-aaaa-bbbbccccdddd',
+        })
+      );
+    });
+
+    const inputsButtons = screen.getAllByRole('button', { name: 'Inputs' });
+
+    for (const inputsButton of inputsButtons) {
+      // Act
+      await user.click(inputsButton);
+
+      screen.getByRole('dialog', { name: 'Inputs' });
+      const table = screen.getByRole('table', { name: 'inputs outputs table' });
+      const rows = within(table).getAllByRole('row');
+      expect(rows.length).toBe(4); // one row for each input definition variable, plus headers
+
+      const headers = within(rows[0]).getAllByRole('columnheader');
+      expect(headers).toHaveLength(2);
+
+      const row1cells = within(rows[1]).getAllByRole('cell');
+      expect(row1cells).toHaveLength(2);
+      expect(row1cells[0]).toHaveTextContent('SRA_ID');
+      expect(row1cells[1]).toHaveTextContent('SRR13379731');
```

Comment:
Nit and not necessary to change, but this test could likely be written slightly more concisely with something like:

```suggestion
      const row1Content = within(rows[1]).getAllByRole('cell').map((el) => el.textContent);
      expect(row1Content).toEqual(['SRA_ID', 'SRR13379731']);
```

Or since a similar test is done several times:

```js
      const getRowContent = (rowIndex) => within(rows[rowIndex]).getAllByRole('cell').map((el) => el.textContent);
      
      expect(getRowContent(1)).toEqual(['SRA_ID', 'SRR13379731']);
      expect(getRowContent(2)).toEqual(...)
```